resources/quality/dagoma_pro_430_bowden/Steel nozzle 0.4/dagoma_pro_430_bowden_0.4_abs_high_quality.inst.cfg

```diff
@@ -0,0 +1,58 @@
+[general]
+
+version = 4
+name = High Quality
+definition = dagoma_pro_430_bowden
+
+[metadata]
+
+setting_version = 19
+type = quality
+quality_type = high_quality
```

Comment:
Nitpicking; Why not just set it to be "high"? Adding the _quality seems a bit double.